styles/Events.module.scss

```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
+  &:hover{
+    color: $white;
+    background: $black;
+    box-shadow: inset 0 0 20px;
+    outline-color: rgba(49, 138, 172, 0);
+    outline-offset: 80px;
```

Comment:
Due to you are using 0 in the alpha channel, outline-color is creating an "invisible" outline, is actually needed to have an outline so? I meant, if it's not visible, maybe can be removed these lines

![lastrecord](https://user-images.githubusercontent.com/66505715/188672407-4baf4de0-9df4-406c-b758-14e434c58c49.gif)